packages/go_router_builder/example/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
   collection: ^1.15.0
   flutter:
     sdk: flutter
-  go_router: ^15.1.1
+  go_router: ^15.2.4
```

Comment:
this should be 16.0.0

Reply:
@chunhtai  I updated it in [Update version number](https://github.com/flutter/packages/pull/9482/commits/770d64ed55efd8c9431576a98792a6a89bb6e397)